v3/src/utilities/translation/lang/en-US.json5

```diff
@@ -205,7 +205,16 @@
     "DG.mainPage.mainPane.versionString": "Version %@ (%@)", // DG.VERSION, DG.BUILD_NUM
     "DG.mainPage.mainPane.messageView.value": "Unfortunately, CODAP is not currently supported on your browser. CODAP is supported on Firefox 46+, Chrome 50+, Windows Edge 14+, and Safari 10+. CODAP is not actively supported on other browsers at this time.",
     "DG.mainPage.titleBar.saved": "Document Saved!",
-    "DG.mainPage.exceptionMessage": "An error has occurred that may affect how this program behaves. You may wish to reload this page after you save your work. (Error: %@)",
```

Comment:
@kswenson do you think it is right to delete this old string? It is replaced by the new strings. And I suppose if I remove it, I should remove it from all of the language files. I'd guess there are lots of strings in this file that are not used in V3. Perhaps I should just leave it, and we can do some cleanup all at once sometime in the future.

Reply:
I'm inclined to leave it for now and plan to do a grand cleanup/synchronization down the road.